src/test/java/kdt/springbootjpa/customer/CustomerRepositoryTest.java

```diff
@@ -0,0 +1,60 @@
+package kdt.springbootjpa.customer;
+
+import kdt.springbootjpa.customer.entity.Customer;
+import kdt.springbootjpa.customer.repository.CustomerRepository;
+import org.junit.jupiter.api.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DataJpaTest
+public class CustomerRepositoryTest {
+
+    @Autowired
+    private CustomerRepository customerRepository;
+
+    @Test
+    @DisplayName("[성공] Customer 저장 및 조회하기")
+    void testSaveAndGetCustomer() {
+        Customer customer = Customer.builder().firstName("tlsdud").lastName("fortune").build();
+
+        customerRepository.save(customer);
+        Customer savedCustomer = customerRepository.findById(customer.getId()).get();
+
+        assertThat(savedCustomer.getFirstName()).isEqualTo(customer.getFirstName());
+        assertThat(savedCustomer.getLastName()).isEqualTo(customer.getLastName());
+        assertThat(customerRepository.findAll().size()).isEqualTo(1);
+    }
+
+    @Test
+    @DisplayName("[성공] Customer 수정하기")
+    void testUpdateCustomer() {
+        Customer customer = Customer.builder().firstName("sinyoung").lastName("bok").build();
+        customerRepository.save(customer);
+
+        String newFirstName = "tlsdud";
+        String newLastName = "fortune";
+        Customer savedCustomer = customerRepository.findById(customer.getId()).get();
+        savedCustomer.changeFirstName(newFirstName);
+        savedCustomer.changeLastName(newLastName);
+
+        Customer updatedCustomer = customerRepository.findById(customer.getId()).get();
+
+        assertThat(updatedCustomer.getFirstName()).isEqualTo(newFirstName);
+        assertThat(updatedCustomer.getLastName()).isEqualTo(newLastName);
```

Comment:
```suggestion
        assertThat(updatedCustomer)
                .hasFieldOrPropertyWithValue("firstName", firstName)
                .hasFieldOrPropertyWithValue("lastName", lastName);
```

한 메서드로 뽑는 것도 좋을거 같습니다 !

Reply:
한 메서드로 뽑는 것도 좋은 것 같습니다!
근데 개인적으로 hasFieldOrPropertyWithValue에 직접 property 이름을 적는 방법이 조금 불안해서요!